server/src/modules/events/mappers/event-dto.ts

```diff
@@ -1,4 +1,16 @@
+import { BasicClub } from "../domain/entities/event";
+
 export interface EventDTO {
```

Comment:
If `event-map.ts` is the only consumer of `EventDTO` I think we should just move this type into `event-map.ts`.

Reply:
after further discussion we decided against this